scripts/requirements-build.txt

```diff
@@ -7,3 +7,4 @@ intelhex
 pylint
 zcbor~=0.8.0
 nrf-regtool~=5.5.1
+windows-curses; sys_platform == 'win32'
```

Comment:
why is this needed here when it's already part of Zephyr requirements ?
https://github.com/nrfconnect/sdk-zephyr/blob/af17fd81e2c823fb38d9487fe8b32051376e7a49/scripts/requirements-base.txt#L37

btw, there's an `e` too much in commit message title: `reequirements`